src/app/admin/create-entity/create-entity.component.html

```diff
@@ -21,7 +21,7 @@
 
 <div id="schemaEntity" class="container-fluid mt-0 ml-12" >
     <div class="row" *ngIf="!isAddFormPg">
-        <div class="col-md-3">
+        <div class="col-md-3" *ngIf="isActive == 'createSchema' || isActive == 'configurations' || isActive == 'create-vc' || isActive == 'ownership' ">
```

Comment:
@suvarnakale Here you can add single condition for test-publish step with (!=)condition

Reply:
yes